docs/styles.md

```diff
@@ -2,14 +2,39 @@
 
 In order to maintain consistency and reliability across all RAISE content, the styling associated with RAISE content is consolidated into a series of css classes located in `src/styles`. 
 
-RAISE Style classes are either written for specific elements, or can be applied to a variety of objects. Those differences are listed below.
+RAISE Style classes are either written for specific elements, or can be applied to a variety of objects. Those differences are listed below. Content Developers 
```

Comment:
I think you started a sentence that wasn't ever completed here :joy: (`Content Developers...`)